brain/openfang_adapter.py

```
@@ -540,60 +577,65 @@ async def sync_config(self) -> bool:
+                                    )
+                                if resp.status_code == 200:
+                                    key_pushed = True
+                                    logger.info("[OpenFang] API key synced to openai fallback")
+                                    break
+                            except Exception:
+                                pass
+
+                    if not key_pushed:
+                        logger.warning("[OpenFang] All key push formats failed, relying on config.toml")
+                else:
+                    # 本地 provider 不需要 key，跳过 key push
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)